sphinx/themes/basic/layout.html

```diff
@@ -153,13 +153,16 @@ <h3>{{ _('Navigation') }}</h3>
 {%- block extrahead %} {% endblock %}
   </head>
   {%- block body_tag %}<body>{% endblock %}
+
+{%- block skip_to_main_content %}<a href="#skip-target">Skip to main content</a>{% endblock %}
+
 {%- block header %}{% endblock %}
 
 {%- block relbar1 %}{{ relbar() }}{% endblock %}
 
 {%- block content %}
   {%- block sidebar1 %} {# possible location for sidebar #} {% endblock %}
-
+    <span id="skip-target"></span>
```

Comment:
Why not just put the id on the document div?

Reply:
Also maybe I would use a more "specific" id like sphinx-main, to make sure it does not clash with another generated id

Reply:
I'm OK with adding the ID to an already existing element in the templates if that's what you all prefer. My original thinking is that if you create a new, empty target, there's a little less risk that someone will accidentally remove/change that element as opposed to the actual main content element.

Reply:
changed this so that the link points directly to main content node